<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="1" colspan="2">
       <widget class="QRadioButton" name="algoRadioBtn">
        <property name="text">
         <string>算法模式</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>选择格式：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" rowspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>选择模式：</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="fileinBtn">
          <property name="text">
           <string>导入图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fileoutBtn">
          <property name="text">
           <string>导出图片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="algoBtn">
          <property name="text">
           <string>处理图像</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="fileRadioBtn">
          <property name="text">
           <string>文件模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_16">
          <property name="text">
           <string>16进制</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_10">
          <property name="text">
           <string>10进制</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="2" colspan="2">
       <widget class="QLineEdit" name="fileInEdit">
        <property name="text">
         <string>lena.bmp</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="rgbCheckBox">
          <property name="text">
           <string>RGB图</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="grayCheckBox">
          <property name="text">
           <string>单色道图</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>选择文件：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
